doc/source/custom_directives.py

```diff
@@ -1261,6 +1261,7 @@ def pregenerate_example_rsts(
 repo_url = "https://github.com/ray-project/ray.git"
 static_dir_name = "_static"
 version_json_filename = "versions.json"
+dereference_suffix = "^{}"
```

Comment:
I missed this change before - do we need these to be defined at the module level and not in the function that uses them? We really don't want to pollute the module namespace if we can avoid it, and as far as I can tell these are only being used in `generate_versions_json`.

Reply:
Will raise a follow up PR to address this since this one got merged. Thanks!